packages/flame_texturepacker/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2021 Blue Fire
```

Comment:
2024? (or is this a symlink?)

Reply:
It's not a symlink since they don't work properly with publishing, but it is a copy of the file.
Doesn't really matter what year that it says there anyway, it's just 'Murican stuff.